### PR TITLE
Add custom annotations feature for all component's PVC

### PR DESCRIPTION
**Pull request by tanuj-soni2**
This PR is followed by issue no.  [1121](https://github.com/goharbor/harbor-helm/issues/1121)

### CODE REVIEW

File: templates/database/database-ss.yaml [hunk 1]
```diff
@@ -140,6 +140,10 @@ spec:
       name: "database-data"
       labels:
 {{ include "harbor.labels" . | indent 8 }}
+      annotations:
+      {{- range $key, $value := $chartmuseum.annotations }}
```
Comment (zyyw) [inline templates/database/database-ss.yaml:144]: $chartmuseum.annotations  => $database.annotations

Comment (tanuj-soni2) [reply templates/database/database-ss.yaml]: Yes I have corrected the changes

**Comment by zyyw** [comment]
hi @tanuj-soni2 thank you for your contribution. Could you please do a squash commit first and then rebase code afterwards?

**Comment by tanuj-soni2** [comment]
@zyyw Please check I have squashed commits

**Comment by zyyw** [comment]
> @zyyw Please check I have squashed commits

Hi @tanuj-soni2 , there are still 8 commits in this PR. Could you please kindly squash them into one commit?
And then rebase (NOT merge) master into your development branch. Thanks again for your effort!

**Review by zyyw — APPROVED** [review]
LGTM

**Comment by tanuj-soni2** [comment]
@ninjadq Could you please approve this MR ?

**Comment by tanuj-soni2** [comment]
@ywk253100 Please approve this MR as it passed all test cases, its is breaking our production

**Review by ywk253100 — APPROVED** [review]

**Comment by tanuj-soni2** [comment]
@ywk253100 Do you have any idea when this will be included in next release?

**Pull request merged**